arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -325,12 +327,20 @@ export class ElectronMainApplication extends TheiaElectronMainApplication {
               devTools: true,
               nativeWindowOpen: true,
               openerId: electronWindow?.webContents.id,
+              webSecurity: false,
```

Comment:
Why `false`? Theia [does not set it to `false`](https://github.com/eclipse-theia/theia/blob/9ab46186f1f4db52b1a4dc42342e43b8c1606aba/packages/core/src/electron-main/electron-main-application.ts#L293-L301). Hence, it's `true` by default.

Reply:
also this one